examples/adaptive_tutorials/adaptive_templates_tutorial_example_2.py

```diff
@@ -0,0 +1,213 @@
+#
+# Set geometry parameters for layer 1:
+
+mc.set_array_variable("UShape_BridgeThickness_Array", 0, 0.5)
+mc.set_array_variable("UShape_Thickness_Inner_Array", 0, 3.5)
+
+mc.set_array_variable("RatioArray_UMagnetWebThickness", 0, 0.14)
+mc.set_array_variable("RatioArray_UMagnetDiameter", 0, 0.3)
+
+# %%
+# Set geometry parameters for layer 2:
+
+mc.set_array_variable("UShape_BridgeThickness_Array", 1, 0.5)
+mc.set_array_variable("UShape_Thickness_Inner_Array", 1, 3.0)
+
+mc.set_array_variable("RatioArray_UMagnetWebThickness", 1, 0.15)
+mc.set_array_variable("RatioArray_UMagnetDiameter", 1, 0.35)
+
+# %%
+# Set geometry parameters for layer 3:
+
+mc.set_array_variable("UShape_BridgeThickness_Array", 2, 0.5)
+mc.set_array_variable("UShape_Thickness_Outer_Array", 2, 1.5)
+mc.set_array_variable("UShape_Thickness_Inner_Array", 2, 2.5)
+
+mc.set_array_variable("RatioArray_UMagnetWebThickness", 2, 0.2)
+mc.set_array_variable("RatioArray_UMagnetDiameter", 2, 0.5)
+
+# %%
+# Set geometry parameters for layer 4:
+
+mc.set_array_variable("UShape_BridgeThickness_Array", 3, 0.5)
+mc.set_array_variable("UShape_Thickness_Outer_Array", 3, 1.0)
+
+mc.set_array_variable("RatioArray_UMagnetWebThickness", 3, 0.35)
+mc.set_array_variable("RatioArray_UMagnetDiameter", 3, 0.8)
+
+# %%
+# Set corner rounding for rotor pockets
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Enable the option to include corner rounding
+# by setting the following to 'Single definition for all':
+
+mc.set_variable("CornerRounding_Rotor", 1)
+
+# %%
+# Set the Rotor Lamination Corner Radius (units are mm):
+
+mc.set_variable("CornerRoundingRadius_Rotor", 0.8)
+
+# %%
+# The curved flux barriers have now been set up
+# for this Synchronous Reluctance Machine example.
+
+# %%
+# Display the example design geometry
+# ------------------------------------
+# Take a screenshot of the geometry that was set up using
+# adaptive templates and save to the working directory.
+#
+# Screenshot the geometry
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+mc.initialise_tab_names()
+mc.save_screen_to_file("Radial", working_folder / "Radial_Geometry_Screenshot.png")
+
+# %%
+# Load, process and display the image
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Load the saved image
+
+img = mpimg.imread(working_folder / "Radial_Geometry_Screenshot.png")
+
+# %%
+# Crop the image to focus on the rotor geometry that was customised using
+# adaptive templates
+
+img_cropped = img[56:341, 250:535, :]
+
+# %%
+# Display the cropped image
+
+imgplot = plt.imshow(img_cropped)
+plt.axis("off")
+plt.show()
+
+# %%
+# The customised rotor geometry is shown for this Synchronous Reluctance Machine
+# example. The flux barriers have been curved using the imported script,
+# and the rotor pocket parameters were adjusted to improve the model.
+
+# %%
+# Exit Motor-CAD
+# --------------
+# Exit Motor-CAD.
+mc.quit()
+
+# %%
+# Remove the temporary working directory
+# shutil.rmtree(working_folder)
```

Comment:
Not quite clear why this is commented? If it's an option for the user to uncomment, it would be worth mentioning this.